proxy/loopback_test.go

```diff
@@ -176,7 +177,7 @@ func TestLoopbackShunt(t *testing.T) {
 			-> <shunt>;
 	`
 
-	testLoopback(t, routes, Params{}, http.StatusTeapot, http.Header{
+	testLoopback(t, routes, Params{OpenTracer: &opentracing.NoopTracer{}}, http.StatusTeapot, http.Header{
```

Comment:
i think that the context should be initialized with the nooptracer when the params don't contain any. That way we wouldn't need to set the nooptracer explicitly everywhere. For the filterstest package, it doesn't matter too much, but for the proxy package it does. See the newContext() "constructure" function, there we could check if the input tracer if nil, then we would set the nooptracer by default.